### PR TITLE
What3Words (Independent Publisher)

### PR DESCRIPTION
**Pull request by mattybeard**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by mattybeard** [comment]
image
image

image

File: independent-publisher-connectors/What3Words/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+                "map": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Map"
+                }
+              }
+            }
+          }
+        },
+        "summary": "ConvertToLatLng",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/What3Words/apiDefinition.swagger.json:230]: @mattybeard Please follow the documentation for all the documentation related issues in the swagger. The summary values are expected to be free from grammatical and spelling errors.
Documentation - [Summaries and Descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/What3Words/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+                          "type": "number",
+                          "format": "float",
+                          "title": "NE Longitude",
+                          "description": "ne-lng"
+                        },
+                        "lat": {
+                          "type": "number",
+                          "format": "float",
+                          "title": "NE Latitude",
+                          "description": ""
```
[the remark this answers is not in the document]
Comment (mattybeard) [reply independent-publisher-connectors/What3Words/apiDefinition.swagger.json]: Added all.

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**